src/lib/components/primitives/message/input-message.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  import T from '$components/_utilities/locale-strings.svelte';
+
+  export let message: string;
+  export let key: string | undefined = undefined;
```

Comment:
non-blocking, but i spent some time trying to find out if there is an equivalent to a `maybe` type in vanilla typescript. there isn't, of course, but we could do something like this:

```suggestion
 type Maybe<T> = T | null | undefined;
 export let key: Maybe<String> = undefined;
```

I'd probably abstract that type out of the file though to reuse it. Is there a signficant value increase in this? probably not outside of if we debate readability of it. So i'll leave this here as a note and see if you like it more or not.

Reply:
I actually like this idea, I can look at updating some of instances of these, but I'd recommend slowly making this change through the whole project as there are likely a lot of instances of this.